src/actions/breakpoints.js

```diff
@@ -36,17 +44,55 @@ async function _getGeneratedLocation(source, sourceMaps, location) {
   return await sourceMaps.getGeneratedLocation(location, source.toJS());
 }
 
+async function syncClientBreakpoint(
+  source,
+  client,
+  sourceMaps,
+  pendingBreakpoint,
+  location
+) {
+  const generatedLocation = await _getGeneratedLocation(
+    source,
+    sourceMaps,
+    location
+  );
+
+  const originalLocation = await sourceMaps.getOriginalLocation(
+    Object.assign({}, generatedLocation, {
+      line: pendingBreakpoint.generatedLocation.line
```

Comment:
yes. and i agree that this needs to be expressed better. I didn't quite finish fixing this

when the source is reloaded, the source id changes. I wasn't sure what the source was so I quickly threw this in to see if it would work. Basically, there is a new connection. This is why we can't delete / find the line after the debuggee is reloaded. 

for example, 
```javascript
pendingBreakpoint.generatedLocation
/* { 
  id: "server1.conn3.child1/34",  
  line: 51234
} */
generatedLocation // 
/* { 
  id: "server1.conn4.child1/34",  
  line: 51235
} */
```

I didn't get around to doing this properly, but we now have the source available in this function, and this should actually be:
```
const originalLocation = await sourceMaps.getOriginalLocation({
  id: source.id,
  line
});
```